src/Catel.MVVM/Catel.MVVM.Shared/Windows/Helpers/WrapControlHelper.cs

```diff
@@ -225,7 +225,29 @@ public static Grid Wrap(FrameworkElement frameworkElement, WrapOptions wrapOptio
                         button.Command = dataWindowButton.Command;
                     }
 
-                    button.Content = dataWindowButton.Text;
+                    if (dataWindowButton.ContentBindingPath != null)
```

Comment:
Yes, I was thinking about it too, but not sure if there are other then text and visibility properties that should be changed during runtime. Other properties that are one time set, can be changed via styles.